From aa560a86c2ad84f7d1ef4b42274bf1408f09ef8d Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.de>
Date: Mon, 18 Nov 2013 13:23:10 +0100
Git-Commit: 9bab24afba2eb663efbd7f681731f8f49d42c4e8
Patch-Mainline: v3.14
References: FATE#316422
Subject: [PATCH 14/21] uhci: compile debugfs conditional on
 CONFIG_DYNAMIC_DEBUG || DEBUG

This makes sure the header files are all there

Signed-off-by: Oliver Neukum <oneukum@suse.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/usb/host/uhci-debug.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/usb/host/uhci-debug.c b/drivers/usb/host/uhci-debug.c
index 4557375..f2ec1f1 100644
--- a/drivers/usb/host/uhci-debug.c
+++ b/drivers/usb/host/uhci-debug.c
@@ -20,7 +20,7 @@
 
 static struct dentry *uhci_debugfs_root;
 
-#ifdef DEBUG
+#if defined(DEBUG) || defined(CONFIG_DYNAMIC_DEBUG)
 
 /* Handle REALLY large printks so we don't overflow buffers */
 static void lprintk(char *buf)
@@ -635,7 +635,7 @@ static const struct file_operations uhci_debug_operations = {
 
 #endif	/* CONFIG_DEBUG_FS */
 
-#else	/* DEBUG */
+#else	/* DEBUG || CONFIG_DYNAMIC_DEBUG*/
 
 static inline void lprintk(char *buf)
 {}
-- 
1.8.4

